plugins/inventory_sync/module/define_player_inventories.lua

```diff
@@ -1,10 +1,11 @@
 local inventories = {
-	defines.inventory.character_main,
-	defines.inventory.character_guns,
-	defines.inventory.character_ammo,
-	defines.inventory.character_armor,
-	defines.inventory.character_vehicle,
```

Comment:
Why the vehicle gone?

Reply:
The "character_vehicle" inventory doesn't actually exist, so there's no point in trying to sync it. If it did exist though, what would it be?

Reply:
Well, technically if the user is in a vehicle? :D But okay, then the question would be why it was there in the first place :D

Reply:
It was there because it was late at night and I just copied over everything that seemed relevant from the API 🤣

Reply:
Well, if the user is in a vehicle we don't want to copy the vehicle inventory, as the vehicle stays behind. But even if you're in a vehicle getting the `charecter_vehicle` inventory still returns nil.